common/global_cpu_executor.cpp

```diff
@@ -23,10 +23,14 @@
 
 #include "wangle/concurrent/CPUThreadPoolExecutor.h"
 
+DEFINE_int32(global_worker_threads, sysconf(_SC_NPROCESSORS_ONLN),
+             "The number of threads for global CPU executor.");
+
```

Comment:
yes i thought about this. is there any case we call GetThreadCounts() outside main?